lib/ansible/modules/cloud/amazon/lambda_info.py

```diff
@@ -341,7 +347,9 @@ def main():
     argument_spec = dict(
         function_name=dict(required=False, default=None, aliases=['function', 'name']),
         query=dict(required=False, choices=['aliases', 'all', 'config', 'mappings', 'policy', 'versions'], default='all'),
-        event_source_arn=dict(required=False, default=None)
+        event_source_arn=dict(required=False, default=None),
+        max_items=dict(),
```

Comment:
Even though these were referenced internally, I don't think we should add them as new options. Boto3's built-in paginators are much more user-friendly. `max_items` and `next_marker` were primarily used for boto-only modules since boto didn't have paginators but it means people have to use Ansible similarly to a programming language to get their expected results. There are built-in paginators for each of the API calls that would use these options https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/lambda.html#paginators.

Reply:
Ah ! Got it